.travis.yml

```diff
@@ -91,7 +91,7 @@ install:
 script:
   - python -OO -c "import xarray"
   - py.test xarray --cov=xarray --cov-config ci/.coveragerc --cov-report term-missing --verbose $EXTRA_FLAGS
-  - git diff upstream/master **/*py | flake8 --diff --exit-zero || true
```

Comment:
Instead of replacing this, I'd actually enforce that the diff must be flake8-clean:

    git diff master **/*py | flake8 --diff

And then add a second check that covers all files but ignores some errors in existing code:

    flake8 --ignore E20,E231,E241,E26,E4,E721,E731,E741 xarray

Which means that the config section can (needs to be) be left as-is.